presctl/cmd/entities/actions.go

```diff
@@ -0,0 +1,81 @@
+package entities
+
+import (
+	"fmt"
+	"net/http"
+	"os"
+	"strings"
+
+	"github.com/spf13/viper"
+)
+
+type httpClient interface {
+	Do(req *http.Request) (*http.Response, error)
+}
+
+// types of possible operations on data
+const (
+	Analyze   = "analyze"
+	Anonymize = "anonymize"
+	Scanner   = "schedule-scanner-cronjob"
+	Stream    = "schedule-streams-job"
+)
+
+// ActionMsg defines the structure of the action post body
+type ActionMsg struct {
+	Text                string
+	AnalyzeTemplateID   string
+	AnonymizeTemplateID string
+}
+
+// restCommand issues an action related REST command to presidio
+func restCommand(httpClient httpClient, url string, contentStr string) *http.Response {
+	req, err := http.NewRequest("POST", url, strings.NewReader(contentStr))
```

Comment:
consider doing all http requests with a retry library. this implementation is not safe.

Reply:
is that common? I mean in other CLIs? 
seems overkill